projects/plugins/jetpack/_inc/client/my-plan/my-plan-header/index.js

```diff
@@ -64,6 +66,8 @@ class MyPlanHeader extends React.Component {
 					isRefundable={ purchase.is_refundable }
 				/>
 			);
+
+			activation = <ProductActivated active={ purchase.active } />;
```

Comment:
I would be totally fine if we replace this with something like:
```
activation = purchase.active ? <ProductActivated /> : null;
```
I don't know what values can `purchase.active` hold, so the check might need to be adjusted according to that.